Kudu.Services.Web/DebugConsole/Default.cshtml

```diff
@@ -184,3 +184,28 @@
 <script src="~/content/scripts/jquery-console/jquery.console.js"></script>
 <script src="~/content/scripts/ace-init.js"></script>
 <script src="~/content/scripts/HelperScript.js"></script>
+
+<script>
+// Old school ASCII spinner
+var i = 0;
+var frames = ['-', '\\', '|', '/'];
+var spinbox = $('div.spinner > .old-school-spinner');
+var timeout = 300;
+
+function spinner() {
+    var i = 0;
+    frames.forEach(function(frame) {
+        setTimeout(function() {
+            spinbox.text(frame);
+        }, timeout * i);
+        i++;
+    });
+
+    setTimeout(function() {
+        spinner();
+    }, timeout * (frames.length));

```

Comment:
why not replace "timeout * (frames.length)" with a actual number, since it won`t change.